lib/Importer/ONNX.cpp

```diff
@@ -399,6 +399,17 @@ void ONNXModelLoader::loadOperator(const onnx::NodeProto &op) {
     return;
   }
 
+  if (typeName == "Squeeze") {
+    auto *in = getOrCreateNodeByName(op.input(0));
```

Comment:
The comment in the code looks a bit odd. Of all things we comment the line that repeats itself in every single operator and not the other stuff?

Reply:
Removed it here. I guess it may due to "copy & paste"...